src/app/view/web/WebChessView.ts

```diff
@@ -21,20 +23,47 @@ export class WebChessView implements ChessBoardView {
     this.viewEventBus.listenOn(eventType, reaction);
   }
 
-  showSelectedPiece(position: { x: number; y: number }): void {}
+  showSelectedPiece(position: Position): void {
```

Comment:
Patrzac co ta metoda robi, to bardziej bym sie sklapnial na nazwe showSelectedSqare. Bo pokazuje kazdy, nie tylko ten gdzie cos stoi.

Reply:
i to jest właśnie błąd w logice, funkcjonalność podświetlania dowolnego pola jest nam kompletnie do niczego nie potrzebna, przy najmniej na ten moment :P muszę to zmienić